### PR TITLE
Remove splunk-auth secret mapping from fedramp environment

### PR DESCRIPTION
**Pull request by gsleeman**

### CODE REVIEW

File: hack/olm-registry/olm-artifacts-template.yaml [hunk 1]
```diff
@@ -251,6 +251,24 @@ objects:
...
+- apiVersion: hive.openshift.io/v1
+  kind: SelectorSyncSet
+  metadata:
+    name: splunk-forwarder-operator-osd-splunk-auth
+    namespace: splunk-forwarder-operator
+  spec:
+    clusterDeploymentSelector:
+      matchExpressions:
+      - { key: api.openshift.com/managed, operator: In, values: ["true"] }
+      - { key: api.openshift.com/environment, operator: In, values: ["production", "staging"] }
```
Comment (dustman9000) [inline hack/olm-registry/olm-artifacts-template.yaml:264]: Can you remove this environment line?
```suggestion
```

File: hack/olm-registry/olm-artifacts-template.yaml [hunk 2]
```diff
@@ -251,6 +251,24 @@ objects:
...
+      - { key: api.openshift.com/noalerts, operator: NotIn, values: ["true"] }
+      - { key: ext-managed.openshift.io/noalerts, operator: NotIn, values: ["true"] }
+      - key: api.openshift.com/legal-entity-id
+        operator: NotIn
+        values: ${{ALERTS_SKIP_LEGALENTITY_IDS}}
+      - { key: hive.openshift.io/cluster-region, operator: NotIn, values: ["cn-north-1", "cn-northwest-1"] }
+      - { key: api.openshift.com/fedramp, operator: DoesNotExist }
     secretMappings:
```
Comment (dustman9000) [inline hack/olm-registry/olm-artifacts-template.yaml:272]: Are secretMappings still needed?

Comment (gsleeman) [reply hack/olm-registry/olm-artifacts-template.yaml:272]: this targets OSD so i think so (although doesn't looks like **secretReferences** are a thing in hive any more)

**Comment by dustman9000** [comment]
@gsleeman - I think we can close this now that we went a different route, correct?

**Comment by gsleeman** [comment]
yes, closing

**Pull request closed without merging**